package.json

```diff
@@ -26,6 +26,9 @@
     "@mapbox/point-geometry": "^0.1.0",
     "@mapbox/togeojson": "^0.16.2",
     "@opendatasoft/api-client": "^0.1.0",
+    "@testing-library/jasmine-dom": "^1.3.3",
```

Comment:
The testing-library dependencies only seem to be used in the tests, so they should be devDependencies I presume?

(I've been trying to move the jasmine type packages to a devDependency in #7489 so we don't increase the size of the TerriaMap image needlessly.)

Reply:
Is that the same issue as https://github.com/TerriaJS/TerriaMap/issues/730, or a separate issue?

Reply:
It is not something caused by recent changes; it has actually existed for quite some time. The proper way to address this would be to have terriajs as a standalone package, built separately from TerriaMap, but I don't have a timeline for how long it will take.

Reply:
I don't understand what you mean by a standalone package, because both terriajs and terriamap seem to be standalone packages already.

The CD failure sounds inconvenient though, is there a log of the failing CD somewhere?

Reply:
@zoran995 I understand you are busy with other things. Is there anything I can do to help with the issue of your CD?